src/helpers/TradeHelper.sol

```diff
@@ -113,6 +113,7 @@ contract TradeHelper is ITradeHelper, ReentrancyGuardUpgradeable, OwnableUpgrade
   event LogSetAdaptiveFeeCalculator(address indexed oldAdaptiveFeeCalculator, address indexed adaptiveFeeCalculator);
   event LogSetOrderbookOracle(address indexed oldOrderbookOracle, address indexed orderbookOracle);
   event LogSetMaxAdaptiveFeeBps(uint32 indexed oldMaxAdaptiveFeeBps, uint32 indexed maxAdaptiveFeeBps);
+  event LogMakerTakerFee(uint256 makerFeeBps, uint256 makerSizeDelta, uint256 takerFeeBps, uint256 takerSizeDelta);
```

Comment:
Should we log absolute fee usd instead of BPS? And should we concern about displaying token amount here?

Reply:
I left absolute fee from that event because it will be derivable with the current data from the event. It saved gas and saved me from compiler stack too deep. Token amount will be displayed on the other event of SettleTradingFeeAmount.